### PR TITLE
Feature/nav bar

### PR DESCRIPTION
**Pull request by aominhlong**
# Description
This PR creates a basic nav-bar including the styling and format of the nav bar for the application. This PR also includes the home button functionality

Were there any issues that arose? 
n/a

Is there anything you need from your teammates? 
n/a

Any other comments, questions, or concerns?

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactor feature (non-breaking change that modifies existing work)
- [ ] Cosmetic changes (changes to layout or visuals in CSS file)

# How Has This Been Tested?
- [ ] React Dev Tools
- [x] Functionality check through web browser
- [ ] Ran Cypress test suite

# Checklist:
- [x] My code follows the style guidelines of this project (semicolons, indentation, naming conventions, etc..).
- [x] I have performed a self-review of my code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have deleted all comments after functionality and understanding is final.
- [x] I have added any issues closed by this PR in the Issues section.
- [x] I have added others as reviewers.

### CODE REVIEW

File: src/Styles/NavBar.css [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    position: absolute;
+    background-color: #f9f9f9;
+    min-width: 160px;
+    z-index: 1;
+}
+
+.dropdown-content > a:hover {
+    border-bottom: 2px solid pink;
+    background-color: #f1f1f1;
+}
```
Comment (rachellhill) [inline src/Styles/NavBar.css:65]: awesome styling!

File: src/Styles/NavBar.css [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    width: auto;
+    text-align: center;
+}
+
+.dropdown:hover > .dropdown-content {
+    display: flex;
+    flex-direction: column;
+}
+
+
```
Comment (stephanie-roe) [inline src/Styles/NavBar.css:78]: great start to our CSS styling!

**Review by rachellhill — APPROVED** [review]
Great work! Excited to get the functionality up and running for the nav bar

**Review by stephanie-roe — APPROVED** [review]
This looks great! Nice work setting up the nav bar

**Review by userigorgithub — APPROVED** [review]
Great job making a new class component NavBar and creating a css for it!

**Pull request merged**